apps/src/templates/sectionAssessments/CorrectAnswerIcon.jsx

```diff
@@ -1,19 +1,54 @@
-import React, {Component} from 'react';
+import React, {Component, PropTypes} from 'react';
 import color from "@cdo/apps/util/color";
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
 
 const styles = {
+  main: {
+    border: 'none',
+    height: '30px',
+    marginTop: '20px',
+    marginRight: '80px',
+    paddingTop: '20px',
+    paddingLeft: '20px',
+    paddingRight: '10px',
+    paddingBottom: '10px',
+  },
   icon: {
     color: color.level_perfect,
-  }
+    float: 'left',
+    width: '10%',
+    fontSize: '25px',
+  },
+  text: {
+    color: color.black,
```

Comment:
our text colour style is `color.charcoal`

Reply:
Thanks, color file is a handy reference.